### ISSUE TITLE
Bitmex amount precision wrong?

### ISSUE THREAD

**Issue by lfern** [ccxt/ccxt#18331]
### Operating System

linux

### Programming Languages

JS

### CCXT Version

3.1.31

### Description

I think the amount precision calc in bitmex is wrong. e.g for BTC/USDT market I get this precision:

```
 "precision": {
      "amount": 10000,
      "price": 0.5,
      "quote": 0.5,
      "base": 0.5
    }
``` 

I have been testing for some symbols and I guess it should be `info['lotSize']/info['underlyingToPositionMultiplier']`.

### Code

I think bitmex code in fetchmarkets function should be 

```
....
                   'precision': {
                        'amount': this.safeNumber(market, 'lotSize') / this.safeNumber(market, 'underlyingToPositionMultiplier'),
                        'price': this.safeNumber(market, 'tickSize'),
                        'quote': this.safeNumber(market, 'tickSize'),
                        'base': this.safeNumber(market, 'tickSize'),
                    },  
....
```

**Comment by carlosmiei** [comment]
Hello @lfern,
You're right Bitmex spot precisions are not unified yet but we have already a PR that fixes this issue https://github.com/ccxt/ccxt/pull/18089, hopefully we will merge it soon, we're just finalizing some tests.

**Comment by carlosmiei** [comment]
@lfern It was merged and it's available as of 3.1.54, can you update your ccxt version and try again pls?
https://github.com/ccxt/ccxt/pull/18089
